napari/_vispy/canvas.py

```diff
@@ -70,26 +198,398 @@ def _set_theme_change(self, theme: str):
         )[0]
         self.bgcolor = self._last_theme_color
 
+    def _disconnect_theme(self) -> None:
+        self.viewer.events.theme.disconnect(self._on_theme_change)
+
     @property
-    def bgcolor(self):
-        SceneCanvas.bgcolor.fget(self)
+    def bgcolor(self) -> str:
+        """Background color of the vispy scene canvas as a hex string. The setter expects str
+        (any in vispy.color.get_color_names) or hex starting with # or a tuple | np.array ({3,4},)
+        with values between 0 and 1."""
+        return self._scene_canvas.bgcolor.hex
 
     @bgcolor.setter
-    def bgcolor(self, value):
-        _value = self._background_color_override or value
-        SceneCanvas.bgcolor.fset(self, _value)
+    def bgcolor(self, value: npt.ArrayLike) -> None:
```

Comment:
base on usage it should also accept strings.

Reply:
resolved with [74d4cbc](https://github.com/napari/napari/pull/5432/commits/74d4cbc129feea9b4ab36230328248480ff76224)